code/modules/surgery/organs/robolimbs.dm

```diff
@@ -3,142 +3,223 @@ GLOBAL_LIST_EMPTY(chargen_robolimbs)
 GLOBAL_LIST_EMPTY(selectable_robolimbs)
 GLOBAL_DATUM(basic_robolimb, /datum/robolimb)
 
+#define model     null
```

Comment:
the use of "null" instead of 0.
but why? 
I added these \#defines based on the comment on like 17 (previously line 13). I was wary of messing stuff up by changing the null to 0, but it still makes me feel uncomfortable. "null, 1, 2"? I wanted to commit "cleaning" this code before working it out, so expect more changes to follow.